std/process.d

```diff
@@ -1266,6 +1266,62 @@ final class Pid
         return _processID;
     }
 
+    version (Windows)
+    {
+        /**
+        A simple factory that returns a $(LREF Pid) handle when provided with a
+        global `pid`. This handle can be used to kill processes that haven't been
+        spawned by this process.
+        Note: this does not check whether the provided `pid` is valid and
+        corresponds to a running process within the operating system.
+
+        Params:
+            pid = a process id
+            desiredAccess = (Windows specific, optional) the desired access to the
+                            process object
+
+        Returns: an instance to a `Pid` that corresponds to the pid within the
+        operating system.
+
+        Throws: $(LREF ProcessException) if opening the process failed; this is
+        Windows specific.
+        */
+        static Pid fromSystemPid(int pid, DWORD desiredAccess = PROCESS_ALL_ACCESS)
+        {
+            import core.sys.windows.winbase : OpenProcess;
+            import std.conv : text;
+            HANDLE _handle = OpenProcess(desiredAccess, false, pid);
+            if (_handle is null)
+            {
+                throw new ProcessException(text("Failed to open process with pid: ", pid));
+            }
+            return new Pid(pid, _handle);
+        }
+    }
+    else version (Posix)
+    {
+        /**
+        A simple factory that returns a $(LREF Pid) handle when provided with a
+        global `pid`. This handle can be used to kill processes that haven't been
+        spawned by this process.
+        Note: this does not check whether the provided `pid` is valid and
+        corresponds to a running process within the operating system.
+
+        Params:
+            pid = a process id
+
+        Returns: an instance to a `Pid` that corresponds to the pid within the
+        operating system.
+
+        Throws: $(LREF ProcessException) if opening the process failed; this is
+        Windows specific.
+        */
```

Comment:
Do not have different documentation blocks for different platforms. Most users will never see the Windows one. Instead, have a look at `version(StdDDoc)`.